### PR TITLE
Made edit contact data into a feature

### PR DESCRIPTION
**Pull request by lagartoverde**
OP-2967

### CODE REVIEW

File: app/routes/administrative-units/administrative-unit/sites/new.js [hunk 1]
```diff
@@ -4,14 +4,20 @@ import {
...
   @service router;
 
+  get editFeature() {
+    const editFeature = config.features['edit-contact-data'];
+    return editFeature === true || editFeature === 'true';
+  }
+
   beforeModel() {
-    if (!this.currentSession.canEdit) {
+    if (!this.currentSession.canEdit || !this.editFeature) {
```
[the remark this answers is not in the document]
Comment (lagartoverde) [reply app/routes/administrative-units/administrative-unit/sites/new.js:20]: Done, didn't know about it thanks

File: app/templates/administrative-units/administrative-unit/core-data/edit.hbs [hunk 2]
```diff
@@ -294,19 +294,20 @@
...
+              (or (not @model.address.id) @model.address.addressRegisterUri)
+              true
+              false
+            }}
+            @primaryContact={{@model.contact}}
+            @secondaryContact={{@model.secondaryContact}}
+          >
+            <:title>Primaire contactgegevens</:title>
+          </Site::ContactEditCard>
+        {{/if}}
```
[the remark this answers is not in the document]
Comment (lagartoverde) [reply app/templates/administrative-units/administrative-unit/core-data/edit.hbs:310]: I misread when I did it, but I read again and I'm not sure what they want, will ask tomorrow in the standup.
Not sure if they want to disable the fields, or just show them as in the show page

File: config/environment.js [hunk 3]
```diff
@@ -77,6 +78,7 @@ module.exports = function (environment) {
   if (environment === 'development') {
     ENV.showAppVersionHash = true;
     ENV.environmentName = 'development';
+    ENV.features['edit-contact-data'] = 'false';
```
[the remark this answers is not in the document]
Comment (lagartoverde) [reply config/environment.js:81]: Not 100% needed but helps with development, as this has to be set in the docker env, and this helps with disabling it without having to run docker

[the remark this answers is not in the document]
Comment (lagartoverde) [reply config/environment.js:81]: Not 100% needed but helps with development, as this has to be set in the docker env, and this helps with disabling it without having to run docker

**Review by aliokan — APPROVED** [review]
Looks good! :+1:

**Pull request merged**